openedx/core/lib/log_utils.py

```diff
@@ -0,0 +1,36 @@
+"""
+Helper functions for logging.
+"""
+import logging
+
+
```

Comment:
oh.  I thought pylint required 2 lines after the imports.  I may have misremembered.  Removed.